app/src/main/res/values/dimens.xml

```diff
@@ -1,49 +1,62 @@
-    <dimen name="font_medium">12sp</dimen>
-    <dimen name="font_medium_more">14sp</dimen>
-    <dimen name="font_medium_extra">16sp</dimen>
-    <dimen name="font_large">18sp</dimen>
-    <dimen name="font_large_extra">20sp</dimen>
-    <dimen name="font_big">22sp</dimen>
-    <dimen name="font_huge">24sp</dimen>
-
-    <!-- Add your custom dimens here -->
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Static Spacing -->
+    <dimen name="static_spacing_tiniest">1dp</dimen>
+    <dimen name="static_spacing_tiny">2dp</dimen>
+    <dimen name="static_spacing_small">4dp</dimen>
+    <dimen name="static_spacing_medium_small">6dp</dimen>
+    <dimen name="static_spacing_medium">8dp</dimen>
+    <dimen name="static_spacing_medium_extra">16dp</dimen>
+    <dimen name="static_spacing_large">20dp</dimen>
+    <dimen name="static_spacing_large_extra">28dp</dimen>
+    <dimen name="static_spacing_big">36dp</dimen>
+    <dimen name="static_spacing_huge">48dp</dimen>
+    <dimen name="static_spacing_huge_extra">60dp</dimen>
+    <dimen name="static_spacing_largest">72dp</dimen>
+
+    <!-- Spacing -->
+    <dimen name="spacing_none">0dp</dimen>
+    <dimen name="spacing_tiniest">1dp</dimen>
+    <dimen name="spacing_tiny">1dp</dimen>
+    <dimen name="spacing_small">2dp</dimen>
+    <dimen name="spacing_medium">4dp</dimen>
+    <dimen name="spacing_medium_more">8dp</dimen>
+    <dimen name="spacing_medium_extra">12dp</dimen>
+    <dimen name="spacing_large">16dp</dimen>
+    <dimen name="spacing_large_extra">22dp</dimen>
+    <dimen name="spacing_large_big">24dp</dimen>
+    <dimen name="spacing_big">28dp</dimen>
+    <dimen name="spacing_big_extra">32dp</dimen>
+    <dimen name="spacing_huge">38dp</dimen>
+    <dimen name="spacing_huge_extra">46dp</dimen>
+    <dimen name="spacing_largest">54dp</dimen>
+    <dimen name="spacing_largest_extra">64dp</dimen>
+
+    <!-- Anti-spacing -->
+    <dimen name="negative_spacing_medium">-4dp</dimen>
+
+    <!-- Text sizes -->
+    <dimen name="font_tiny">6sp</dimen>
+    <dimen name="font_small">8sp</dimen>
+    <dimen name="font_small_more">10sp</dimen>
+    <dimen name="font_medium">12sp</dimen>
+    <dimen name="font_medium_more">14sp</dimen>
+    <dimen name="font_medium_extra">16sp</dimen>
+    <dimen name="font_large">18sp</dimen>
+    <dimen name="font_large_extra">20sp</dimen>
+    <dimen name="font_big">22sp</dimen>
+    <dimen name="font_huge">24sp</dimen>
+
+    <!-- Layout -->
+    <dimen name="cover_width">430dp</dimen>
+    <dimen name="cover_height">195dp</dimen>
+    <dimen name="logo_height">160dp</dimen>
+    <dimen name="logo_width">228dp</dimen>
+    <dimen name="logo_margin_top">100dp</dimen>
+    <dimen name="field_height">56dp</dimen>
+    <dimen name="email_margin_top">260dp</dimen>
+    <dimen name="field_margin_top">2dp</dimen>
+    <dimen name="login_margin_top">15dp</dimen>
+    <dimen name="signup_margin_bottom">100dp</dimen>

```

Comment:
When you use custom resources, like these ones from in dimen, but also in strings resources, is a good practice to use a "label" like you did using `<!-- Layout -->` but is better if the label refers to the layout or the "theme" that is going to using this resources. For example, your "label" could be `<!-- Login -->` and your dimens name should be:
```
<!-- Login -->
<dimen name="login_cover_width">430dp</dimen>
<dimen name="login_cover_height">195dp</dimen>
<dimen name="login_logo_height">160dp</dimen>
...
```